cfme/tests/networks/provider/test_events.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Network entities (e.g. subnets)."""
+import pytest
+
+from cfme.networks.provider.nuage import NuageProvider, NetworkProvider
+
+
+pytestmark = [
+    pytest.mark.provider([NetworkProvider], scope="module")
+]
+
+
+@pytest.yield_fixture
+def prerequisites(provider):
+    provider.teardown_test_entities_on_provider()
```

Comment:
Question: 2 provider.teardown_test_entities_on_provider(), I assume here should be a setup